tic-tac-toe/src/scripts/calculate-next-move.js

```diff
@@ -0,0 +1,91 @@
+function checkTurn(arr, own, opossit) {
```

Comment:
What is opossit? What is own? Name them clearly.

Reply:
I tried to fix this.
```suggestion
function checkTurn(arr, mineValue, rivalValue) {
```

Reply:
These names aren't good either.